llvm/lib/Target/X86/X86.td

```diff
@@ -1779,6 +1775,10 @@ def : ProcModel<"alderlake", AlderlakePModel,
 // FIXME: Use Gracemont Schedule Model when it is ready.
 def : ProcModel<"gracemont", AlderlakePModel,
                 ProcessorFeatures.ADLFeatures, ProcessorFeatures.GRTTuning>;
+foreach P = ["sierraforest", "grandridge"] in {
+  def : ProcModel<P, AlderlakePModel, ProcessorFeatures.SRFFeatures,
```

Comment:
shoud we change `AlderlakePModel` into `GRTModel`? If so, the title needs modified.

Reply:
I think not. There's no GRTModel and the Gracemont is using AlderlakePModel as well.

I think we can improve it in the future, but not need to do with this patch.